table.html

```diff
@@ -7,26 +7,42 @@
 </head>
 <body>
 
-<div>
+
+
     <form>
+
         <h2>Vložte prosím vstup pro formulář:</h2>
+
+        <div class="container">
+    <div>
         <label for="first_name">Jméno: </label>
         <input type="text" minlength="1" maxlength="30" name="first_name" id="first_name" required>
-
+    </div>
+    <div>
         <label for="last_name">Příjmení: </label>
         <input type="text" minlength="1" maxlength="30" name="last_name" id="last_name" required>
+    </div>
 
+    <div class="email">
         <label for="email">Email: </label>
         <input type="email" id="email" name="email" required>
+    </div>
 
+    <div>
         <label for="phone">Telefon: </label>
-        <input type="tel" name="phone" id="phone" required><br><br>
+        <input type="tel" name="phone" id="phone" required>
+    </div>
+
+        <div>
+            <button type="button" onclick="addRow()">Přidej řádek</button>
+        </div>
+
```

Comment:
tady ti chybi jedno zaviraci `</div>` pro zavreni elementu z radku 16 (`<div class="container">`). Melo by to byt pred zaviracim `</form>`